client-v2/src/actions/ExternalArticles.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import type { Action } from 'types/Action';
+
+function errorReceivingArticles(error: string): Action {
+  return {
+    type: 'CAUGHT_ERROR',
+    message: 'Could not fetch articles from Capi',
```

Comment:
What's the difference between 'error' and 'message' here. Might be better to leave rendering a generic message to the view layer?

Reply:
Good question! I've ported any error handling from the original project here; there's definitely another conversation to have about error handling in a more generic sense, which hopefully will be another PR in the not-too-distant future, if that makes sense.

Reply:
👍